### PR TITLE
mktplinkfw2: Add 16Mmtk layout

### PR DESCRIPTION
**Pull request by zamaudio**
This supports eg, TP-Link Archer MR200 v6

See new pending port: https://github.com/openwrt/openwrt/pull/15610

Signed-off-by: Damien Zammit <damien@zamaudio.com>

### CODE REVIEW

File: src/mktplinkfw2.c [hunk 1]
```diff
@@ -149,6 +149,12 @@ static struct flash_layout layouts[] = {
 		.kernel_la	= 0x80000000,
 		.kernel_ep	= 0x80000000,
 		.rootfs_ofs	= 0x140000,
+	}, {
+		.id		= "16Mmtk",
+		.fw_max_len	= 0xf90000,
```
Comment (DragonBluep) [inline src/mktplinkfw2.c:154]:
```suggestion
		.fw_max_len	= 0xfa0000,
```

Comment (zamaudio) [reply src/mktplinkfw2.c:154]: Aha, yes I miscalculated by 64kB.

Comment (Ansuel) [reply src/mktplinkfw2.c]: @DragonBluep mhhh the other 16m entry are f9... why this should be fa?

Comment (DragonBluep) [reply src/mktplinkfw2.c]: This is `firmware` partition size. See @zamaudio PR https://github.com/openwrt/openwrt/pull/15610/files. ath79 target has different partition layout.

**Pull request merged**